elastic-apm-agent/pom.xml

```diff
@@ -31,6 +31,14 @@
                             <createSourcesJar>true</createSourcesJar>
                             <shadeSourcesContent>true</shadeSourcesContent>
                             <shadedArtifactAttached>false</shadedArtifactAttached>
+                            <filters>
+                                <filter>
+                                    <artifact>*:*</artifact>
+                                    <excludes>
+                                        <exclude>META-INF/versions/*/module-info.class</exclude>
```

Comment:
I think this structure for locating the module-info class is specific to multi-release jars. Maybe `*/module-info.class` or even `*module-info.class` will be more inclusive, or did you aim for only such jars?

Reply:
done